real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/sensors/HillCrestImuNode.java

```diff
@@ -228,4 +278,18 @@ public double convertQNToDouble(byte lsb, byte msb, int qn) {
 
     }
 
+    /**
+     * Returns a 2-byte data value from the IMU's data array
+     * @param data the byte array for the IMU
+     * @param offsetInBytes the offset into the data array where the halfword starts from. Note that this value does
+     *                      NOT get increased, you will need to keep track of an offset externally
```

Comment:
maybe not, but I think it's good to err on the side of caution. This is sort of a menial thing to keep track of, so it'll be easily forgotten. Any reminder that we have is one less bug to fix